ui/raidboss/data/06-ew/trial/endsinger-ex.ts

```diff
@@ -734,9 +734,22 @@ const triggerSet: TriggerSet<Data> = {
         'The Endsinger': '讴歌终结之物',
       },
       'replaceText': {
+        'Befoulment': '脓液弹',
+        'Benevolence': '博爱',
+        'Despair Unforgotten': '绝望侵蚀：记录事件',
+        'Diairesis': '分离',
+        'Eironeia': '反讽',
+        'Elegeia Unforgotten': '哀歌：记录事件',
         'Elenchos': '反诘',
+        'Endsong\'s Aporrhoia': '流溢：绝望轮唱',
+        'Endsong(?!\')': '绝望轮唱',
         'Fatalism': '宿命',
```

Comment:
```suggestion
        '(?<! )Fatalism': '宿命',
```

Because `Theological Fatalism` and `Fatalism` try to translate the same phrase, you need to differentiate them like this.